fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingMultiTargetByRecords.java

```diff
@@ -112,17 +111,12 @@ List<Object> indexingLogMessageKeyValues() {
     CompletableFuture<Void> buildIndexInternalAsync() {
         return getRunner().runAsync(context -> openRecordStore(context)
                 .thenCompose( store -> context.getReadVersionAsync()
-                            .thenCompose(vignore -> {
-                                SubspaceProvider subspaceProvider = common.getRecordStoreBuilder().getSubspaceProvider();
-                                // validation checks, if any, will be performed here
```

Comment:
Was this comment intended to be that:
> buildMultiTargetIndex will do any necessary validation checks

Or, the more problematic for this change:
> getSubspaceAsync will do any necessary validation checks

? I'm pretty sure the answer is that `buildMultiTargetIndex` is doing the validation checks, but wanted to confirm.

Reply:
This comment appeared in the indexers as a hint/reminder where to add validations. I think that appears because `IndexingByIndex`  had a source/target indexes validations at the post `context.getReadVersionAsync` lambda, and then its structure was copied by the other indexers. I did not think that this comment is needed anymore.